src/simplay/src/simplay/components.py

```diff
@@ -21,28 +18,50 @@ class VisualProcess(VisualComponent):
 
     :param env: The environment instance.
     :param id: The id of the component.
-    :param graphic: The graphic of the component, either a 'simple' visual or a collection of sprites. Must be registered in the ``VisualizationManager``.
-    :param tint: The tint of the component. This only works with visuals and sprites that have transparent pixels. The tint is applied to the pixels that are not transparent.
+    :param graphic: The graphic of the component, either a 'simple' visual or
+     a collection of sprites. Must be registered in the
+     `VisualizationManager`.
+    :param tint: The tint of the component. This only works with visuals and
+     sprites that have transparent pixels. The tint is applied to the pixels
+     that are not transparent.
     """
 
-    def __init__(self, env: VisualEnvironment, id: str, graphic: str, tint: int):
+    def __init__(
+            self,
+            env: VisualEnvironment,
+            id: str,
+            graphic: str,
+            tint: int):
         super().__init__(env, id, "PROCESS", graphic, tint)
```

Comment:
Can we use a constant instead of "PROCESS"
Please check the whole source code for that there are a lot of magic strings if possible we should have them in constants and also use them in the tests

e.g. if I search for "Id must be a string." on this page I get 23 findings